dktest.go

```diff
@@ -167,6 +176,13 @@ func waitContainerReady(ctx context.Context, lgr logger, c ContainerInfo,
 
 // Run runs the given test function once the specified Docker image is running in a container
 func Run(t *testing.T, imgName string, opts Options, testFunc func(*testing.T, ContainerInfo)) {
+	RunT(t, imgName, opts, func(_ TestingT, containerInfo ContainerInfo) {
+		testFunc(t, containerInfo)
+	})
+}
+
+// RunT is the same as Run, but uses a testing.TB interface instead of testing.T.
```

Comment:
Update the comment to mention using the `dktest.TestingT` interface instead of the `testing.TB` interface